c/csrc/evalspv.c

```diff
@@ -0,0 +1,98 @@
+#include "stdbool.h"  // CKB override
+#include "stdint.h"   // CKB override
+#include "stdio.h"
+#include "stdlib.h"  // CKB override
```

Comment:
Personally I don't really recommend mixing the minimal libc in ckb-c-stdlib with another libc. You will either want the libc in ckb-c-stdlib, or use the libc that comes with GNU toolchain. Mixing both together might have unexpected results.

Reply:
yeah, you're right. stdio isn't even used. pushing a commit that cleans up the includes